package.json

```diff
@@ -1,55 +1,38 @@
 {
-  "author": "Jared Allard <jaredallard@outlook.com>",
   "name": "nexe",
-  "description": "create single executables out of your [node/io].js applications",
+  "description": "Create a single executable out of your Node.js application",
   "license": "MIT",
-  "version": "1.1.2",
+  "version": "2.0.0-beta.1",
   "contributors": [
-    {
-      "name": "Criag Condon",
-      "email": "craig.j.condon@gmail.com",
-      "url": "http://crcn.io/"
-    }
+    "Craig Condon <craig.j.condon@gmail.com> (http://crcn.io)",
+    "Jared Allard <jaredallard@outlook.com>",
+    "Caleb Boyd <caleb.boyd@hotmail.com>"
   ],
   "scripts": {
-    "test": "echo node: $(node -v) && node_modules/.bin/mocha --reporter mocha-circleci-reporter test/test.js"
+    "lint": "eslint index.js ./src/**/*",
+    "test": "npm run lint"
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/jaredallard/nexe.git"
+    "url": "git://github.com/nexe/nexe.git"
+  },
+  "main": "src/nexe.js",
+  "bin": {
+    "nexe": "index.js"
   },
-  "main": "./lib/index.js",
   "dependencies": {
-    "async": "^1.5.2",
-    "browserify": "^13.0.0",
-    "colors": "^1.1.2",
-    "glob": "^7.0.0",
-    "gunzip-maybe": "^1.3.1",
-    "insert-module-globals": "^7.0.1",
+    "app-builder": "^5.1.0",
+    "bluebird": "^3.5.0",
+    "chalk": "^1.1.3",
+    "memory-fs": "^0.4.1",
+    "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
-    "module-deps": "^4.0.5",
-    "ncp": "^2.0.0",
-    "progress": "^1.1.8",
-    "request": "^2.67.0",
-    "tar-stream": "^1.3.1",
-    "yargs": "^4.2.0"
-  },
-  "nexe": {
-    "input": "./bin/nexe",
-    "output": "nexe^$",
-    "temp": "src",
-    "runtime": {
-      "framework": "nodejs",
-      "version": "5.5.0",
-      "ignoreFlags": true
-    }
-  },
-  "preferGlobal": true,
-  "bin": {
-    "nexe": "bin/nexe"
+    "request": "^2.81.0",
+    "rimraf": "^2.6.1",
+    "tar": "^2.2.1",
+    "webpack": "^2.3.3"
   },
   "devDependencies": {
-    "mocha": "^2.4.5",
-    "mocha-circleci-reporter": "0.0.1"
+    "eslint": "^3.19.0"
```

Comment:
How do you feel about transpiling this?   Generators are gross, but `async`/`await` would make this much "cleaner", IMO.  I'd be happy to do the work to add babel if it would help.

Also, what are your feeling on just bringing n the [airbnb JS standards](https://github.com/airbnb/javascript)/eslint config?   It would be a standard, and this PR wouldn't have to include that marathon `.eslintrc`